mesonbuild/compilers.py

```diff
@@ -2729,3 +2733,135 @@ def unix_compile_flags_to_native(self, args):
 
     def get_link_debugfile_args(self, targetfile):
         return []
+
+
+# Only tested on linux for ICC 17.0.0
+#    * object extraction does not compile (don't know) (25 object extraction fails)
+#    * ICC returns no error if not existing flags are used (i.e. -fsomeflag),
```

Comment:
No, I mean does ICC emit warnings about unknown flags, or does it at least have a flag to enable warnings if there isn't one for erroring out on unknown flags?